apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx

```diff
@@ -72,6 +73,13 @@ function TransferBody({ op }: { op: TransferOpEvent }) {
   const [account] = useAccount();
   assert(account != null);
 
+  const opRequestId = op.nonceMetadata
+    ? DaimoNonceMetadata.fromHex(op.nonceMetadata).identifier.toString()
+    : undefined;
+  const matchingTrackedRequest = account.trackedRequests.find(
+    (req) => req.requestId === opRequestId && req.amount === `${op.amount}`
```

Comment:
might be worth also requiring that the recipient is us

Reply:
done